app/Http/Middleware/LogActivityMiddleware.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use App\Services\DebugRequestActivityLogger;
+use App\Services\DummyRequestActivityLogger;
+use App\Services\ProductionRequestActivityLogger;
+use Closure;
+use Illuminate\Http\Request;
+
+class LogActivityMiddleware
+{
+    private DummyRequestActivityLogger $logger;
+    private DebugRequestActivityLogger $debug_logger;
```

Comment:
Won't be accepted. It was not requested.

Reply:
changed